### ISSUE TITLE
Per-Hero Settings Feature Request

### ISSUE THREAD

**Issue by Neuffexx** [blockout22/OverwatchVisualScripting#5]
I am trying to Port my game mode into the scriptor, but it heavily relies on a specific hero being available.
I noticed that on the Left hand menu there is a hero settings section, but that only encompases the `All` hero settings.

It would be great if settings for All heros ( + enable / disable toggles) would be exposed.
Unless they already are available somewhere else and I just havent found it yet?

**Comment by blockout22** [comment]
I've added those changes for you a880696 which lets you toggle which heroes should be enabled or disabled

**Comment by Neuffexx** [comment]
Great thank you.
Additionally, would it be possible to add the hero unique settings?
I am not sure how consistent overwatch is with the options for each hero and therefore how easy this would be to do, but it would be great if this could be added in a manner similarily to the Mode specific settings:
  

![image_2024-06-13_132103812](https://github.com/blockout22/OverwatchVisualScripting/assets/63469833/1f0bbc9a-b16d-44b2-af4a-2eaf694453ff)

![javaw_gPONKDrVLD](https://github.com/blockout22/OverwatchVisualScripting/assets/63469833/a94b5c04-5bae-4363-8356-0b2ba73e7984)

![image](https://github.com/blockout22/OverwatchVisualScripting/assets/63469833/61494bee-e0e2-414f-ba37-2a464705a803)

**Comment by blockout22** [comment]
Yeah it's easy enough to add but would it be worth adding since all those hero settings can be changed in the workshop scripts?

**Comment by Neuffexx** [comment]
True enough, it was simply a suggestion to be able to edit everything within the same place.
